app/main.py

```diff
@@ -1 +1,61 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive: list["Animal"] = []
+
+    def __init__(
+        self: "Animal",
+        name: str,
+        health: int = 100,
+    ) -> None:
```

Comment:
```suggestion
class Animal:
    alive: list[Animal] = []

    def __init__(
        self:
        name: str,
        health: int = 100,
    ) -> None:
```

You don’t need to specify a type annotation for `self`.